app/lib/actions/katello/repository_set/disable_repository.rb

```diff
@@ -18,8 +18,8 @@ def humanized_name
           _("Disable")
         end
 
-        def plan(product, content, substitutions)
-          if repository = repository_mapper(product, content, substitutions).find_repository
+        def plan(product, content, substitutions, registry_name = nil)
```

Comment:
I'm liking #2 better but I'm not fond of the name `options` for the hash. Sounds too generic.